osu.Game/Screens/Play/Player.cs

```diff
@@ -450,6 +451,12 @@ private Drawable createOverlayComponents(IWorkingBeatmap working)
                 Children = new[]
                 {
                     DimmableStoryboard.OverlayLayerContainer.CreateProxy(),
+                    new LetterboxOverlay
+                    {
+                        Alpha = working.Beatmap.LetterboxInBreaks ? 1 : 0,
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                    },
```

Comment:
Huh? This makes the letterbox overlay *ever-present* if the beatmap setting is toggled on, no? `LetterboxOverlay` has no internal break-related logic. It's only ever supposed to show *during breaks*.

Reply:
😯

Reply:
Fixed this and also made the visual of this match stable, because what we had felt pretty bad to my eye.